index.html

```diff
@@ -10,16 +10,49 @@
     <link href="https://fonts.googleapis.com/css2?family=Comic+Neue:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap" rel="stylesheet">
 </head>
 <body>
-    <h1>Awesome-books</h1>
-    <table id="list-books">
+    <header class="header">
+        <a href="#list" id="logo">Awesome Books</a>
+        <nav id="navbar">
+            <ul id="contain-nav-item">
+                <li id="nav-list" class="nav-item">
+                    <a id="nav-list-a" class="nav-item-a" href="#list">List</a>
+                </li>
+                <li id="nav-add-new" class="nav-item">
+                    <a id="nav-add-new-a" class="nav-item-a" href="#add-new">Add new</a>
+                </li>
+                <li id="nav-contact" class="nav-item">
+                    <a id="nav-contact-a" class="nav-item-a" href="#contac">Contact</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+     <section id="list" class="list">
+        <h1>Awesome-books</h1>
+        <table id="list-books">
+      
+        </table>
+     </section>      
+     <section id="add-new" class="hiden add-new">
+        <h2>Add a new book</h2>
+        <form id="form" action="">
+            <input type="text" id="title"  class="from-control" name="title" placeholder="Title" required>
+            <input type="text" id="author"  name="author" placeholder="Author" required>
+            <button type="submit" id="add" placeholder="Add">Add</button>
+        </form>
+    </section>
+    <section class="contact hiden" id="contact">
+       <h2 class="contact-title">Contact information</h2>
+          <p class="contact-p">Do have any question or you just went to say "Hello"?<br> You can reach out to us! </p>
+            <ul class="contact-a">
+                <li class="contact-l">Oure-mail: mail@.com</li>
+                <li class="contact-l">Our phone number: 0043586534422</li>
+                <li class="contact-l">Our address: Streetname:22,84503 City, Country </li>
+            </ul>
+    </section>
 
-    </table>
-    <h2>Add a new book</h2>
-    <form id="form" action="">
-        <input type="text" id="title"  class="from-control" name="title" placeholder="Title" required>
-        <input type="text" id="author"  name="author" placeholder="Author" required>
-        <button type="submit" id="add" placeholder="Add">Add</button>
-    </form>
+    <footer class="footer">
+        <h4 class="footer-h">Copyright ...</h4>
+    </footer>
```

Comment:
Good job adding a `footer` to your page. 👍🏿  However, there is one suggestion I would want to make to better your page. ♻️ 

- [ ] Kindly make sure your `footer` is at the foot of the page. ⬇️ Currently your `footer` hangs inside your page. as seen below 🔽 
![image](https://github.com/newtonah0/Awesome-books/assets/117144028/7d712ba5-f59c-4ba8-b5a1-6f0b03f98cb3)